_articles/building-the-best-one-click-sign-in.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: article
+title: "Building The Best One Click Sign In"
+abstract: "One click sign in using device authentication offers convenience and security to the user."
+sitemap:
+  lastmod: 2020-05-18
+---
+
+One click sign in allows users to authenticate in just one click.  Delivering this was the reason DID.app was started. This is our story of implementing one click sign in.
+
+## Why do we need one click sign in?
+
+Users have ‘account creation fatigue’. The average person already has 150+ accounts and the problem is still getting worse. There are so many websites that want you to create an account and it's hard to keep track of them all.
+
+Users want to engage without filling in endless forms, choosing yet another password or confirming their email address.  Users want convenience.  Or, to put it another way, users are lazy; I know I am.
+
+## How does one click sign in work?
+
+Single factor authentication can relies on either: something you know, something you have or something you are.  In a typical username and password setup, the user relies on something only they know (their password) to prove their identity.  For one click sign in, we rely on something the user has.  The one thing every user has is a device. 
+
+The user authenticates by proving they have access to a specific device. If the device is present, the user is authenticated.  This is achieved with a simple cryptographic transaction between DID.app and the user’s device initiated by a single click.
+
+Users should be familiar with this type of authentication. It works on the same principle as using as a key to unlock your front door or using debit card to pay for something.  These are both forms of authentication that rely on something the users has (a key or a debit card) and are used multiple times every day.
+
+## Speed, Security and Safeguards
+
+We set ourselves the challenge of making One Click Sign In really fast and more secure with sensible safeguarding for fallback.
+
+
+This server to server communication is at the mercy of the network connection between the device, the website and the social network.
+
+
+DID.app's One Click Sign In works using cryptography.  A key pair generated and stored by the device are challenged by DID.app to prove identity.  The user actually has to have access to the physical device for One Click Sign In to work.  Users can set up multiple devices if they regularly use more than one device.  This greatly reduces the threat of account theft because hacking is only economical on scale and without actually having to steal a person's handbag.
```

Comment:
I think this belongs after "This is achieved with a simple cryptographic transaction" in the how it works section